src/Orchard.Web/Modules/Orchard.OutputCache/Filters/OutputCacheFilter.cs

```diff
@@ -267,45 +268,55 @@ public class OutputCacheFilter : FilterProvider, IActionFilter, IResultFilter {
 
         protected virtual bool RequestIsCacheable(ActionExecutingContext filterContext) {
 
+            var url = filterContext.RequestContext.HttpContext.Request.RawUrl;

```

Comment:
Please wrap this code by testing if `Logger.IsEnabled(LogLevel.Debug)` is true, then build `itemDescriptor`.